auth.go

```diff
@@ -50,12 +53,16 @@ func newAuthAddr(addr net.Addr) string {
 // Auth stores lookups for bans, whitelists, and ops. It implements the sshd.Auth interface.
 // If the contained passphrase is not empty, it complements a whitelist.
 type Auth struct {
-	passphraseHash []byte
-	bannedAddr     *set.Set
-	bannedClient   *set.Set
-	banned         *set.Set
-	whitelist      *set.Set
-	ops            *set.Set
+	passphraseHash  []byte
+	whitelistModeMu sync.RWMutex
```

Comment:
While we're at it, would you be up for renaming as much of our code from `whitelist` to `allowlist` please? Seems like a good time to do it. :)